apps/web/ui/links/use-link-filters.tsx

```
@@ -293,3 +345,107 @@ function useUserFilterOptions({ folderId }: { folderId: string }) {
+  {
+    value: "searchEngineIndexing",
+    label: "Search Engine Indexing",
+    icon: WindowSearch,
+  },
+] as const;
+
+function useLinkFeatureFilterOptions({ folderId }: { folderId: string }) {
+  const { showArchived } = useContext(LinksDisplayContext);
+
+  const counts = FEATURE_OPTIONS.map((feature) =>
+    useLinksCount<number>({
```

Biome (2.1.2)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)